rl/api/endpoints.py

```diff
@@ -36,6 +111,26 @@ def config():
         key2 - bla bla
     """
     if request.method == "PUT":
+        if runner.running:
+            response = flask.jsonify(
+                {"error": "Stop training/testing before changing configuration"}
+            )
+            response.status_code = 400
+            return response
+
+        data = json.loads(request.data)
+        algorithm_manager.update_config(data)

```

Comment:
Warto byłoby dodać tutaj walidację pt. zwróć błąd jeżeli parametr "niemodyfikowalny" został zmodyfikowany.

Wiem, że taka sytuacja nie powinna się zdarzyć, ale zawsze to jest dodatkowy bufor bezpieczeństwa na wypadek gdyby jakimś sposobem został zmodyfikowany parametr "niemodyfikowalny"

Reply:
ok zrobione